metadata_service/entity/resource_type.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum, auto
+
+
+class ResourceType(Enum):
```

Comment:
may put a comment on why we don't put user here given user is considered a resource entity in Amundsen.

Reply:
Mmm... Actually, I can put `User` here as well. Will do that.